src/app/components/create-button/create-button.component.html

```diff
@@ -0,0 +1,4 @@
+<button (click)="onClick()">
+  <i class="fa-duotone fa-plus"></i>
+  Create
```

Comment:
Let's make the button reusable by using `ng-content` for content projection. In that case, the name `create-button` also has to change as it'll act as a generic button then.

Reply:
done